aider.el

```diff
@@ -381,6 +402,78 @@ The command will be formatted as \"/ask \" followed by the text from the selecte
   :lighter " Aider"
   :keymap aider-minor-mode-map)
 
+(defvar aider-prog-mode-map (make-sparse-keymap)
+  "Keymap for Aider commands in programming modes.")
+
+;;;###autoload
+(define-minor-mode aider-prog-mode
```

Comment:
Thanks for other changes. I am not very sure about this minor mode. What is the benefit to define a minor mode here?
For existing aider-minor-mode, I feel it is very experimental, it is for a aider command dedicate file. (eg. maybe sth like a command.aider, but everything inside is aider command, for reference)

Reply:
I don't think this is being limited correctly anyway, and I think it is not needed considering the other direction I mentioned in the other comment.